cypress/support/TestListManager.js

```diff
@@ -142,5 +143,5 @@ function GetTestGroup(name) {
 }
 
 function GetTest(testGroup, testNameToFind) {
```

Comment:
Still think it's weird to have what looks like a `Get` method internally be a `find`.  Do we expect this method to ever not find a test?  If not, then I would change this to `throw` an error if `find` returns `undefined`

Like:
```

Reply:
I've never really considered the subtle difference between a GET and FIND method, I've always lumped them into the same category. So again I thank you for continuing to educate me on such things.

I changed those two methods (GetTestGroup too) to reflect this way of thinking about it.

Reply:
Hmm, I had commented on this previously here and you had even replied so I thought I would bring it up again.
https://github.com/Microsoft/ConversationLearner-UI/pull/953#discussion_r251109416

Thanks for changing it. Even if you don't agree, javascript is written with thinking so want to keep consistent.